src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/CreateRoom.java

```diff
@@ -85,42 +90,56 @@ public void react(ReceivedMessage parsedMessage, Messenger messenger, Database d
 
             JsonObject response = new JsonObject();
             response.addProperty("messageContextId", parsedMessage.getMessageContextId());
-            response.addProperty("type", "CreateRoomResponse");
-            response.addProperty("result", "OK");
+            response.addProperty("type", ResponseType.CREATE_ROOM_RESPONSE.toString());
```

Comment:
So this `ResponseType.CREATE_ROOM_RESPONSE.toString()` will result in `CREATE_ROOM_RESPONSE` string?

Reply:
That's right. There is an option to write custom .toString() function which will return `CreateRoomResponse` String but in my opinion it's better to transfer `type` excatly as ENUM looks like. Same thing on ClientSide. It increases readability.

Reply:
Currently client has to write type of request the same as the name of the class extending `Reaction`. So if we want change convention into upper case snake case, then i have to change framework a little to make an option to specify different name than the class name or... and that's ridiculous, we have to write class names extending reaction like CREATE_ROOM 🤣.

If you want, i can change the framework a little bit to make custom names possible. Actually i did something like this earlier, but i forgo it.

Reply:
So right now when user is listening for response from the server, his code should contain a class named `CREATE_ROOM_RESPONSE`? Am i thinking right?

Reply:
No no no. Currently user can have classes whatever it wants to.

If user sends request to the server with type `CREATE_ROOM`, then server has to have reaction class named `CREATE_ROOM`.